src/app/(search)/search/page.tsx

```diff
@@ -10,7 +10,19 @@ import { isSearchSystemReleased } from "@/constants/FeatureFlag.constants";
 
 export const metadata: Metadata = {
   title: "Arman Ahmadi - Search",
+  description: "Here you can search about articles you want to read with Arman",
```

Comment:
Change to the following:
"Search articles by Arman for topics you want to read."

Reply:
Done